### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Jalixdev** [community/community#40131] [Other Feature Feedback, Questions, & Ideas]

**Comment by Preetiraj3697** [comment]
Hello @Jalixdev. How are you. welcome to great community.

**Comment by backlogAK** [comment]
Hello @Jalixdev

**Comment by ossanna16** [comment]
Hi there @Jalixdev and welcome to our community! 

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
